### DISCUSSION TITLE
Donut or Sunburst chart examples?

### DISCUSSION THREAD

**Discussion by dgwyer** [observablehq/plot#1007] [Ideas]
I've looked through the docs and examples but I can't seem to find any examples of pie, donut, or sunburst charts. Is it easily possible to create these types of charts with Observable Plot?

**Comment by Fil** [comment]
Not yet. See issue https://github.com/observablehq/plot/issues/133 (and https://github.com/observablehq/plot/pull/85 for an early prototype).

**Comment by dgwyer** [comment]
Oh, OK. Thanks.

Would be great to get this added to the main library.

**Comment by bMorgan01** [comment]
I apologize for reviving this.

Any updates? Timeline? I see @Fil's draft PR, but that hasn't been touched since March.

**Reply by Fil** [reply]
No timeline, but you can vote the issue up, and share examples of what you'd like to achieve—this will help us prioritize.

**Comment by deepaktammali** [comment]
Hello,

Thanks for the library. Is there a way to build donut/pie charts using Plot? I searched in the docs website couldn't find one.

**Reply by mbostock** [reply]
See the comments above for status and links to the relevant feature requests.
